src/transformers/models/pegasus/modeling_flax_pegasus.py

```diff
@@ -692,7 +686,6 @@ def setup(self):
             self.config.max_position_embeddings, embed_dim, dtype=self.dtype
         )
         self.layers = FlaxPegasusEncoderLayerCollection(self.config, self.dtype)
-        self.layer_norm = nn.LayerNorm(dtype=self.dtype)
```

Comment:
This `layer_norm` should not be deleted, Pegasus has one `layer_norm` in both encoder and decoder which is applied to the `last_hidden_state`

https://github.com/huggingface/transformers/blob/1c191efc3abc391072ff0094a8108459bc08e3fa/src/transformers/models/pegasus/modeling_pegasus.py#L780-L783

https://github.com/huggingface/transformers/blob/1c191efc3abc391072ff0094a8108459bc08e3fa/src/transformers/models/pegasus/modeling_pegasus.py#L1047-L1049